app/src/main/AndroidManifest.xml

```diff
@@ -19,7 +19,9 @@
             android:name=".MainActivity"
             android:exported="true"
             android:label="@string/title_activity_main"
-            android:theme="@style/Theme.HikeMate">
+            android:theme="@style/Theme.HikeMate"
+            android:screenOrientation="portrait"
+            tools:ignore="DiscouragedApi,LockedOrientationActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
 
```

Comment:
Fix: Usage of such method is "Discouraged" (as written in your ignore file). Have you explored using `setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);` in the MainActivity?

Reply:
Mmm might work better for the tests too, I'll give it a try tomorrow !

Reply:
Done, however it does not help with the test, since the test itself sets the `requestedOrientation` of the activity. I'm not sure there's another way to test this.